lib/SPIRV/SPIRVRegularizeLLVM.cpp

```diff
@@ -154,6 +156,34 @@ SPIRVRegularizeLLVM::regularize() {
         }
       }
     }
+
```

Comment:
Please refactor this part as a member function and add come comments about how it works.

I guess this only fixes the specific issue for switch instruction, not a generic transformation to guarantee that blocks appearing before all blocks they dominate, right? If so it is OK, just mention that in the comment and give it a proper name e.g. fixBlockOrderForSwitch.